modules/openapi-generator/src/main/resources/swift4/CodableHelper.mustache

```diff
@@ -10,62 +10,66 @@ import Foundation
 public typealias EncodeResult = (data: Data?, error: Error?)
 
 open class CodableHelper {
+    
+    private static var customDateFormatter: DateFormatter?
+    private static var defaultDateFormatter: DateFormatter = {
+        let dateFormatter = DateFormatter()
+        dateFormatter.calendar = Calendar(identifier: .iso8601)
+        dateFormatter.locale = Locale(identifier: "en_US_POSIX")
+        dateFormatter.timeZone = TimeZone(secondsFromGMT: 0)
+        dateFormatter.dateFormat = Configuration.dateFormat
+        return dateFormatter
+    }()
+    private static var customJSONDecoder: JSONDecoder?
+    private static var defaultJSONDecoder: JSONDecoder = {
+        let decoder = JSONDecoder()
+        decoder.dateDecodingStrategy = .formatted(CodableHelper.dateFormatter)
+        return decoder
+    }()
+    private static var customJSONEncoder: JSONEncoder?
+    private static var defaultJSONEncoder: JSONEncoder = {
+       let encoder = JSONEncoder()
+        encoder.dateEncodingStrategy = .formatted(CodableHelper.dateFormatter)
+        encoder.outputFormatting = .prettyPrinted
+        return encoder
+    }()
```

Comment:
What do you think about moving those configurations to a central place where they will be more visible?
https://github.com/OpenAPITools/openapi-generator/blob/775cf655f99acbd54d1f9865fdcdab29a46e3cd7/modules/openapi-generator/src/main/resources/swift4/APIs.mustache#L9-L15
This way all the configurations are in the same place.

Reply:
On the other hand, all the configurations that are present in APIs.mustache are related to API requests... This is more about parsing, so maybe here may be the right place, but it's hard to discover this configuration.
Whats are your thought on this @pcoltau and @wing328?